### PR TITLE
Add .NET SDK MGMT Tests for Resources

### PR DESCRIPTION
**Pull request by HarveyLink**
Add .NET SDK MGMT Tests for Resources

### CODE REVIEW

File: common/ManagementTestShared/ManagementRecordedTestBase.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 using Azure.Core;
 using Azure.Core.TestFramework;
-using Azure.Management.Resources;
+using Azure.ResourceManager.Resources;
```
Comment (erich-wang) [inline common/ManagementTestShared/ManagementRecordedTestBase.cs:7]: We can't simply change in this way as it is shared by multiple projects. Please change as below

```suggestion
#if RESOURCES_RP
using Azure.ResourceManager.Resources;
#else
using Azure.Management.Resources;
#endif
```

File: sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj [hunk 2]
```diff
@@ -1,5 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TestHelperProjects>Rbac1.6;</TestHelperProjects>
```
Comment (erich-wang) [inline sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj]: Please add
```
    <DefineConstants>$(DefineConstants);RESOURCES_RP</DefineConstants>
```

**Review by erich-wang — CHANGES_REQUESTED** [review]

File: sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj (hunk 2 above)
Comment (erich-wang) [inline sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj]: If Rbac1.6 is not used, please change to:
```
<TestHelperProjects>;</TestHelperProjects>
```

File: sdk/resources/Azure.ResourceManager.Resources/tests/ResourceOperationsTestsBase.cs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            ResourcesClient = ResourcesManagementClient.GetResourcesClient();
+            TagsClient = ResourcesManagementClient.GetTagsClient();
+            SubscriptionsClient = ResourcesManagementClient.GetSubscriptionsClient();
+            TenantsClient = ResourcesManagementClient.GetTenantsClient();
+            PolicyAssignmentsClient = ResourcesManagementClient.GetPolicyAssignmentsClient();
+            PolicyDefinitionsClient = ResourcesManagementClient.GetPolicyDefinitionsClient();
+            PolicySetDefinitionsClient = ResourcesManagementClient.GetPolicySetDefinitionsClient();
+        }
+
+        internal ResourcesManagementClient GetResourcesManagementClient()
```
Comment (erich-wang) [inline sdk/resources/Azure.ResourceManager.Resources/tests/ResourceOperationsTestsBase.cs]: Should we reuse the one in base class

File: common/ManagementTestShared/ManagementRecordedTestBase.cs (hunk 1 above)
Comment (HarveyLink) [reply common/ManagementTestShared/ManagementRecordedTestBase.cs:7]: Done

File: sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj (hunk 2 above)
Comment (HarveyLink) [reply sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj]: Done

File: sdk/resources/Azure.ResourceManager.Resources/tests/ResourceOperationsTestsBase.cs (hunk 3 above)
Comment (HarveyLink) [reply sdk/resources/Azure.ResourceManager.Resources/tests/ResourceOperationsTestsBase.cs]: Done

File: sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj (hunk 2 above)
Comment (HarveyLink) [reply sdk/resources/Azure.ResourceManager.Resources/tests/Azure.ResourceManager.Resources.Tests.csproj]: Done

**Pull request merged**